Nos/Views/NoteComposer/NoteComposer.swift

```diff
@@ -85,18 +92,15 @@ struct NoteComposer: View {
                                         }
                                     }
                                 }
+                                .id(editingController)
```

Comment:
This interacts with `.disabled(showNotePreview)`. Without the line that disables the note composer, the user can still edit the note while displaying the preview. With that line, we prevent the user from editing while previewing. However, while in macOS that works perfectly, in iOS the preview never loads, it shows the preview briefly and then an infinite spinner wheel. I don't know why exactly that happens, I suspect it is because some object is destroyed by SwiftUI and the preview cannot find the attributedString or something, but I cannot explain it for sure. I tried different approaches, like trying with `resignFirstResponder() `in UIKit, but nothing worked, so I ended adding this line here that prevents SwiftUI from rebuilding what's inside unless `editingController` changes. I didn't find anything wrong with it, not a refression, but it is definitely something we need to carefully inspect for side changes.